image-loader/test/scala/model/ProjectorTest.scala

```diff
@@ -33,7 +34,9 @@ class ProjectorTest extends FunSuite with Matchers with ScalaFutures with Mockit
   private val config = ImageUploadOpsCfg(new File("/tmp"), 256, 85d, Nil, "img-bucket", "thumb-bucket")
 
   private val s3 = mock[AmazonS3]
-  private val projector = new Projector(config, s3, imageOperations)
+
+  // TODO MRB: put these behind an interface at the top level so we can mock them here
+  private val projector = new Projector(config, s3, imageOperations, null, null)
```

Comment:
The actual test here is ignored, it was presumably useful when we built the projection service during the missing images work

Reply:
hmm 😬